src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,8 @@
+.container {
```

Comment:
Please check, should be 4 cards per row
![image](https://user-images.githubusercontent.com/54936044/230998196-d7f172b6-b9a5-4350-bc04-45c7493ff859.png)
![image](https://user-images.githubusercontent.com/54936044/230998336-4dacac56-2416-4004-a88a-0e084d2ab08c.png)